### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RanvirChoudhary**
## Description
Removed the redis link from the database integration table

**Fixes** #(issue)

This fixes issue #6212 

I have read the CLA Document and I hereby sign the CLA

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,8 +128,7 @@ MindsDB works with most of the SQL and NoSQL databases, data lakes, data Streams
...
+| <a href="https://docs.mindsdb.com/data-integrations/questdb"><img src="https://img.shields.io/badge/QuestDB-d14671?style=for-the-badge&logo=questdb&logoColor=white" alt="Connect QuestDB"></a> | <a href="https://docs.mindsdb.com/data-integrations/apache-druid"><img src="https://img.shields.io/badge/Apache%20Druid-29F1FB?logo=apachedruid&logoColor=000&style=for-the-badge" alt="Apache Druid Badge"></a> | <a href="https://docs.mindsdb.com/data-integrations/starrocks"><img src="https://img.shields.io/badge/starrocks-000000?style=for-the-badge&logo=starrocks&logoColor=white" alt="StarRocks Badge"></a> | <a href="https://docs.mindsdb.com/data-integrations/vertica"><img src="https://img.shields.io/badge/Vertica-000?logo=vertica&logoColor=fff&style=for-the-badge" alt="Vertica Badge"></a> | <a href="https://docs.mindsdb.com/data-integrations/duckdb"><img src="https://img.shields.io/badge/duckdb-fff?logo=duckdb&logoColor=000&style=for-the-badge" alt="DuckDB Badge"></a> |
```
Comment (martyna-mindsdb) [inline README.md:131]: You merged lines 131 and 132 such that now one line contains 5 tags.

Please correct it to have one line with 2 tags and another line with 3 tags. For example:

line 131:
```
| <a href="https://docs.mindsdb.com/data-integrations/questdb"><img src="https://img.shields.io/badge/QuestDB-d14671?style=for-the-badge&logo=questdb&logoColor=white" alt="Connect QuestDB"></a> | <a href="https://docs.mindsdb.com/data-integrations/apache-druid"><img src="https://img.shields.io/badge/Apache%20Druid-29F1FB?logo=apachedruid&logoColor=000&style=for-the-badge" alt="Apache Druid Badge"></a> | <a href="https://docs.mindsdb.com/data-integrations/starrocks"><img src="https://img.shields.io/badge/starrocks-000000?style=for-the-badge&logo=starrocks&logoColor=white" alt="StarRocks Badge"></a> |
```

line 132:
```
| <a href="https://docs.mindsdb.com/data-integrations/vertica"><img src="https://img.shields.io/badge/Vertica-000?logo=vertica&logoColor=fff&style=for-the-badge" alt="Vertica Badge"></a> | <a href="https://docs.mindsdb.com/data-integrations/duckdb"><img src="https://img.shields.io/badge/duckdb-fff?logo=duckdb&logoColor=000&style=for-the-badge" alt="DuckDB Badge"></a> |
```

**Comment by martyna-mindsdb** [comment]
recheckcla

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

**Comment by martyna-mindsdb** [comment]
@RanvirChoudhary 
Please sign the CLA as mentioned above.

Also, have a look at the review comment and push updates to this PR.

**Comment by RanvirChoudhary** [comment]
@martyna-mindsdb i have updated the pull request with the mentioned changes and edited the original pull request message to include the CLA signature, please check.

**Comment by martyna-mindsdb** [comment]
@RanvirChoudhary 
To sign the CLA, please leave a comment here that contains only this line:

I have read the CLA Document and I hereby sign the CLA

**Comment by RanvirChoudhary** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by martyna-mindsdb** [comment]
recheckcla

**Pull request merged**